acceptance/config_test.go

```diff
@@ -18,14 +18,14 @@ const configFilename = "test.toml"
 
 type TestConfig struct {
 	// Place to describe what's wrong with this test. Does not affect how the test is run.
-	Badness string
+	Badness *string
 
 	// Which OSes the test is enabled on. Each string is compared against runtime.GOOS.
 	// If absent, default to true.
 	GOOS map[string]bool
```

Comment:
Should it be the same for map values here?

Reply:
I don't think so, maps already distinguish between true/false and absent.